lib/vat_info.rb

```diff
@@ -0,0 +1,32 @@
+require 'vat_info/version'
+require 'vat_info/utils'
+require 'vat_info/request'
+require 'vat_info/response'
+require 'vat_info/query'
+require 'vat_info/models'
+
+module VatInfo
+  def self.unreliable_payer(*vat_ids)
+    request  = VatInfo::Request::UnreliablePayer.new(*vat_ids).to_xml
+    response = VatInfo::Query.call(request, :get_status_nespolehlivy_platce)
+
+    if response.ok?
+      status_raw    = response.raw[:status_nespolehlivy_platce_response][:status]
```

Comment:
Nešlo by přesunout celou tu inicializaci response do jejího initialize? Takhle je to různě rozházený v kódu a chvíli mi trvalo se v tom vyznat.

Reply:
zatim teda nic, kdyžtak to refaktoruju v další verzi